### PR TITLE
DOC-2376 Update the Cloud API Quickstart for January sprint

### PR DESCRIPTION
**Pull request by ianjevans**
New reference doc, and updated quickstart with new message responses.

### CODE REVIEW

**Review by erademacher — CHANGES_REQUESTED** [review]
Cluster structure changes look good! I think it would still be helpful to include a section about the GetNodes endpoint, now that node status isn't included in the GetCluster results.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12846#-:-MuMmthSAk5QggUzXWyC:b-bmphth)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ianjevans, @marksoper, and @stbof)

___
*[cockroachcloud/cloud-api.md, line 223 at r1](https://reviewable.io/reviews/cockroachdb/docs/12846#-MuMmLxKC5dC4wYQeS6Y:-MuMmLxLDGJzi_19HUHI:b80ipi5) ([raw file](https://github.com/cockroachdb/docs/blob/45432bec1b5413ab971a0e4d1925892dd7a57696/cockroachcloud/cloud-api.md#L223)):*
> ```Markdown
>   - `{region name}` is the cloud infrastructure provider region where the cluster is located.
>   - `{spend limit}` is the [maximum amount of money, in US cents, you want to spend per month](serverless-cluster-management.html#planning-your-cluster) on this cluster.
>   - `{node name}` is the name of the node in the cluster.
> ```

Users can still check node status using the clusters/{id}/nodes endpoint. Can you add a section about that, please?

File: cockroachcloud/cloud-api.md [hunk 1]
```diff
@@ -117,38 +117,47 @@ If the request was successful, the API will return information about the newly c
...
+  - `{clusterId}` is the unique ID of this cluster. Use this ID when making API requests for this particular cluster.
     {{site.data.alerts.callout_info}}
-    The cluster ID used in the Cloud API is different than the cluster name and tenant ID used when [connecting to clusters](connect-to-a-serverless-cluster.html).
+    The cluster ID used in the Cloud API is different than the routing ID used when [connecting to clusters](connect-to-a-serverless-cluster.html).
     {{site.data.alerts.end}}
   - `{cluster name}` is the name of the cluster you specified when creating the cluster.
-  - `{CockroachDB version}` is the version of CockroachDB running on this cluster.
   - `{account ID}` is the ID of the account that created the cluster. If the cluster was created using the API, this will be the service account ID associated with the secret key used when creating the cluster.
   - `{region name}` is the zone code of the cloud infrastructure provider where the cluster is located.
+  - `{routingId}` is the cluster name and tenant ID of the cluster used when [connecting to clusters](connect-to-a-serverless-cluster.html).
```
Comment (stbof) [inline cockroachcloud/cloud-api.md:159]: It's hard to picture what the routingID looks like with this definition. Is there some reason it's not the same as it appears here: http://www.cockroachlabs.com/docs/cockroachcloud/connect-to-a-serverless-cluster.html#step-2-connect-to-your-cluster  

`identifies your tenant cluster on a multi-tenant host. For example, funny-skunk-123.`

File: cockroachcloud/cloud-api.md [hunk 2]
```diff
@@ -174,54 +183,48 @@ If the request was successful, the API will return detailed information about th
...
   {{site.data.alerts.end}}
   - `{cluster name}` is the name of the cluster you specified when creating the cluster.
-  - `{CockroachDB version}` is the version of CockroachDB running on this cluster.
   - `{cloud provider}` is the name of the cloud infrastructure provider on which you want your cluster to run. Possible values are: `GCP` and `AWS`. The default value is `GCP`.
   - `{account ID}` is the ID of the account that created the cluster. If the cluster was created using the API, this will be the service account ID associated with the secret key used when creating the cluster.
   - `{region name}` is the cloud infrastructure provider region where the cluster is located.
   - `{spend limit}` is the [maximum amount of money, in US cents, you want to spend per month](serverless-cluster-management.html#planning-your-cluster) on this cluster.
-  - `{node name}` is the name of the node in the cluster.
-  - `{node status}` is the status of the node. Possible values are: `LIVE` and `NOT_READY`
+  - `{routingId}` is the cluster name and tenant ID of the cluster used when [connecting to clusters](connect-to-a-serverless-cluster.html).
```
Comment (stbof) [inline cockroachcloud/cloud-api.md:226]: same comment as line 159.

File: cockroachcloud/cloud-api.md [hunk 3]
```diff
@@ -117,38 +117,47 @@ If the request was successful, the API will return information about the newly c
...
-  - `{clusterId}` is the unique ID of this cluster. Use this ID when making API requests for this particular cluster. Note:
+  - `{cloud provider}` is the name of the cloud infrastructure provider on which you want your cluster to run. Possible values are: `GCP` and `AWS`. The default value is `GCP`.
+  - `{clusterId}` is the unique ID of this cluster. Use this ID when making API requests for this particular cluster.
     {{site.data.alerts.callout_info}}
-    The cluster ID used in the Cloud API is different than the cluster name and tenant ID used when [connecting to clusters](connect-to-a-serverless-cluster.html).
+    The cluster ID used in the Cloud API is different than the routing ID used when [connecting to clusters](connect-to-a-serverless-cluster.html).
     {{site.data.alerts.end}}
   - `{cluster name}` is the name of the cluster you specified when creating the cluster.
-  - `{CockroachDB version}` is the version of CockroachDB running on this cluster.
   - `{account ID}` is the ID of the account that created the cluster. If the cluster was created using the API, this will be the service account ID associated with the secret key used when creating the cluster.
```
Comment (stbof) [inline cockroachcloud/cloud-api.md:157]: The convention for variable names should be consistent. Why is it `{clusterId}` and `{routingId}`, but {account ID}?

**Review by stbof — CHANGES_REQUESTED** [review]
Please address questions.

**Comment by ianjevans** [comment]
> Cluster structure changes look good! I think it would still be helpful to include a section about the GetNodes endpoint, now that node status isn't included in the GetCluster results.
...
> Users can still check node status using the clusters/{id}/nodes endpoint. Can you add a section about that, please?

Ok, I'll add a section on `ListClusterNodes`.

**Review by stbof — APPROVED** [review]

**Review by erademacher — APPROVED** [review]
:lgtm: 
Thanks!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12846#-:-MuNHWgC2EzTi26XgY6u:b-911c3n)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ianjevans and @marksoper)

**Pull request merged**